flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/StandaloneResourceManagerFactory.java

```diff
@@ -54,6 +57,14 @@
 			highAvailabilityServices,
 			rpcService.getScheduledExecutor());
 
+		final Time clusterInitTime;
+		if (configuration.contains(ResourceManagerOptions.CLUSTER_INITIALIZATION_TIME) &&
+			configuration.getLong(ResourceManagerOptions.CLUSTER_INITIALIZATION_TIME) > 0) {
+			clusterInitTime = Time.milliseconds(configuration.getLong(ResourceManagerOptions.CLUSTER_INITIALIZATION_TIME));
+		} else {
+			clusterInitTime = Time.milliseconds(configuration.getLong(JobManagerOptions.SLOT_REQUEST_TIMEOUT));
+		}
+
```

Comment:
Paring of clusterInitTime occurs in both `StandaloneResourceManagerFactory` and `StandaloneResourceManagerWithUUIDFactory`, I think it can be moved to `ResourceManagerRuntimeServicesConfiguration`